### DISCUSSION TITLE
Where is the code that pushes to https://hub.docker.com/r/summerwind/actions-runner/tags?

### DISCUSSION THREAD

**Discussion by cep21** [actions/actions-runner-controller#1468] [Questions]
I'm trying to find the dockerfile.  It doesn't look like this one: https://github.com/actions-runner-controller/actions-runner-controller/blob/master/Dockerfile

**Comment by cep21** [comment]
I see, it's moved to https://github.com/actions-runner-controller/actions-runner-controller/blob/master/runner/actions-runner.dockerfile

**Reply by toast-gear** [reply]
https://github.com/actions-runner-controller/actions-runner-controller/blob/master/Dockerfile = controller image
https://github.com/actions-runner-controller/actions-runner-controller/blob/master/runner/actions-runner.dockerfile = runner image for the default runner configuration
https://github.com/actions-runner-controller/actions-runner-controller/blob/master/runner/actions-runner-dind.dockerfile = the alterantive runner image for the [all in one runner](https://github.com/actions-runner-controller/actions-runner-controller#runner-with-dind)
